src/library/general_library.lua

```diff
@@ -526,5 +526,37 @@ function library.calc_script_name(include_extension)
     return retval
 end
 
+--[[
+% get_used_font_name
+
+Processes an input font name into Mac or Windows specific versions. This function is used by get_defualt_music_font().
+
+@ [standard_name] (string) The name of the font as returned by FCFontInfo:GetName().
+: (string) The OS-specific name of the font.
+]]
+function get_used_font_name(standard_name)
+    local font_name = standard_name
+    if string.find(os.tmpname(), "/") then
+        font_name = standard_name
+    elseif string.find(os.tmpname(), "\\") then
+        font_name = string.gsub(standard_name, "%s", "")
+    end
+    return font_name
+end
+
+--[[
+% get_default_music_font
+
```

Comment:
I would prefer this function be called `calc_default_music_font_name`. That makes it clear that what is returned is a string and not an `FCFontInfo` instance.  Also, see my question at `get_used_font_name`. We may need to add a prefix like `os`, e.g., `calc_default_music_font_os_name`. Note the use of the `calc` prefix. We should be migrating towards using the `get` prefix for getting properties while `calc` is used for calculating values that are not properties.

Reply:
Great, will do!